app/src/main/java/com/example/admin/showdata/View_Holder.java

```diff
@@ -0,0 +1,23 @@
+package com.example.admin.showdata;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+/**
+ * Created by admin on 15-02-2018.
+ */
+
+public class View_Holder extends RecyclerView.ViewHolder {
```

Comment:
Create the view-holder class in the same adapter class instead of creating a new file. This stops from flooding the directories with files.